### DISCUSSION TITLE
scfinder outputs in the xml file

### DISCUSSION THREAD

**Discussion by dufangqing** [SED-EEW/SED-EEW-SeisComP-contributions#55] [Q&A] [answered]
Dear SED-EEW community, 

I was wondering how to save scfinder outputs to a xml file. I was able to do it simply by redirecting the stdout to a file, but this only applies to the `--playback` `--offline` mode.

For `msrtsimul -c *.miniseed | seiscomp-finder exec scfinder -I -`, how can i obtain the xml file for the outputs?

Thanks a lot,
Du

**Comment by FMassin** [comment, marked as answer]
The option to do that is not implemented sorry. But you can dump the latest event from database using scevtls and scxmldump.

**Reply by dufangqing** [reply]
Thanks very much！
